src/58MP.yaml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Flow mapping edge cases
+  from: '@ingydotnet'
+  tags: edge flow mapping
+  yaml: |
+    {"x"
+    : y}
```

Comment:
It might not be a good idea to use a plain `y` because 1.1 loaders will resolve that as a bool, and it will appear as a fail (diff), although syntactically they can parse it.
I think we should avoid such scalars; schema testing should be an extra thing.

Reply:
7ba5479e